### PR TITLE
chore: frontend formatting

### PR DESCRIPTION
**Pull request by cbe**
As discussed this draft is a suggestion to format frontend files to prevent a mix-up of tabs/spaces. Sadly c7ad3ffd42a599c86d8977d7058ee02660c340c3 is a rather large commit due to the configuration forcing streamlined line endings (and other things like indentation) to all less & js files.
I would understand if this is too large of a changeset for such a minor thing.

### CODE REVIEW

**Comment by fmartingr** [comment]
This so far seems good to me, I don't mind if changes are big if we stick to only one formatting. Sadly github does not render tab/spaces changes apparently, but took a quick glance and everything seems good.

**Review by fmartingr — APPROVED** [review]
This looks good to me, thanks for adding the check to the script as well! I'm going to request @Monirzadeh's requests as well, to have another set of eyes.

Seems weird to me that some files show full file changes when everyhing seems the same. Did the CRLF change as well?

File: scripts/styles.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ OUTPUT_ARCHIVECSS=internal/view/assets/css/archive.css
 # Use bun is installled
 if [ -x "$(command -v bun)" ]; then
     bun install
+    bun x prettier internal/ --write
```
[the remark this answers is not in the document]
Comment (cbe) [reply scripts/styles.sh]: Good point, changed it!

**Comment by fmartingr** [comment]
Awsome work, let's put this in :)

**Pull request merged**